Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -93,6 +97,55 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
         }
     }
 
+    private func createEmptyStateOverlayView() -> UIView {
+        let overlayView = UIView()
+        overlayView.backgroundColor = UIColor.whiteColor()
+
+        let logoImageView = UIImageView(image: UIImage(named: "emptyBookmarks"))
+        overlayView.addSubview(logoImageView)
+        logoImageView.snp_makeConstraints { make in
+            make.centerX.equalTo(overlayView)
+
+            // Checks if the device is an iPad
+            if traitCollection.horizontalSizeClass == .Regular && traitCollection.verticalSizeClass == .Regular {
+                make.centerY.equalTo(overlayView.snp_centerY).offset(HomePanelUX.EmptyTabContentOffset)
+            } else {
+                make.top.greaterThanOrEqualTo(overlayView.snp_top).offset(50)
```

Comment:
Nit: Don't think you need greaterThan here. Just equals should be fine since there isn't any other conflict constraints.